lib/active_merchant/billing/gateways/micropayment.rb

```diff
@@ -159,8 +163,9 @@ def split_authorization(authorization)
         authorization.split("|")
       end
 
-      def authorization_from(response)
-        "#{response["sessionId"]}|#{response["transactionId"]}"
+      def authorization_from(response, request_params)
+        session_id = response["sessionId"] ? response["sessionId"] : request_params[:sessionId]
```

Comment:
Looks like there are some responses that don't have a `session_id`, eh?  This makes me :sob:.